File: src/oneline/1945/S1945.py
class Solution:
    def getLucky(self, s: str, k: int) -> int:
        return reduce(lambda x, _: sum(map(lambda z: int(z), str(x))), range(k-1), sum(map(sum, map(lambda z: divmod(ord(z)-96, 10), s))))

File: src/oneline/1945/S1945.java
class Solution {
    public int getLucky(String s, int k) {
        return IntStream.range(0, k-1).reduce(s.chars().map(a->a-96).map(b->b/10+b%10).sum(), (x, y)->Integer.toString(x).chars().map(c->c-48).sum());
    }
}
